test/cpp/qps/stats.h

```diff
@@ -43,7 +43,7 @@ namespace testing {
 template <class T, class F>
 double sum(const T& container, F functor) {
   double r = 0;
-  for (auto v = container.begin(); v != container.end(); v++) {
+  for (auto v = container.begin(); v != container.end(); ++v) {
```

Comment:
So... it was to make it consistent within the file with changes I decided not to make... I'll revert it... it was only accidentally pulled in.